### PR TITLE
[MINOR] improve(docs): update the doc of the web

### PR DESCRIPTION
**Pull request by ch3yne**
### What changes were proposed in this pull request?

Add description for creating fileset and messaging catalog, and add some tips.

### Why are the changes needed?

Fix: #2100

### Does this PR introduce _any_ user-facing change?

N/A

### How was this patch tested?

N/A

### CODE REVIEW

File: docs/webui.md [hunk 1]
```diff
@@ -225,25 +233,55 @@ Creating a catalog requires these fields:
...
+    <Image img={require('./assets/webui/create-fileset-hadoop-catalog-dialog.png')} style={{ width: 480 }} />
+
+  </TabItem>
+</Tabs>
+
+###### 3. Type `messaging`
+
+<Tabs>
+  <TabItem value='kafka' label='Kafka'>
+    Follow the [Kafka catalog](./kafka-catalog.md) document.
```
Comment (ch3yne) [inline docs/webui.md:255]: It seems there are currently no Kafka-related documents, so we need to wait for the related PR to be merged.

File: docs/webui.md [hunk 2]
```diff
@@ -52,13 +48,20 @@ gravitino.authenticator = oauth
 
 Set the configuration parameter `gravitino.authenticator` to `oauth`, and the web UI displays the login page.
 
-![webui-login-with-oauth](assets/webui/login-with-oauth.png)
+:::caution
+If both `OAuth` and `HTTPS` are set, due to the different security permission rules of various browsers, to avoid cross-domain errors,
+it is recommended to use the [Chrome](https://www.google.com/chrome/) browser for access and operation.
+
+Such as Safari need to enable the developer menu, and select `Disable Cross-Origin Restrictions` from the develop menu.
+:::
```
Comment (ch3yne) [inline docs/webui.md:56]: Related issue #2100

File: docs/webui.md [hunk 3]
```diff
@@ -268,33 +306,31 @@ Clicking on the action icon <Icon icon='mdi:delete-outline' fontSize='24' color=
...
 Displays a confirmation dialog, clicking on the SUBMIT button deletes this catalog.
 
-![delete-catalog](assets/webui/delete-catalog.png)
+![delete-catalog](./assets/webui/delete-catalog.png)
 
 ### Schema
 
-Under Construction...
+Design draft preview:
+
```
Comment (jerryshao) [inline docs/webui.md:314]: I think you don't have to mention "Design draft preview", you can delete it here and below.

File: docs/webui.md [hunk 4]
```diff
@@ -268,33 +306,31 @@ Clicking on the action icon <Icon icon='mdi:delete-outline' fontSize='24' color=
...
 Design draft preview:
 
-![demo-tables](assets/webui/demo-tables.png)
+![demo-columns](./assets/webui/demo-columns.png)
 
-### Table
+### Fileset
 
-Under Construction...
+Preview:
```
Comment (jerryshao) [inline docs/webui.md:325]: You can remove the "Preview" here and below.

**Review by jerryshao — APPROVED** [review]

**Pull request merged**